src/components/comments/Comment.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext, useRef } from "react"
+import { CommentContext } from "./CommentProvider"
+
+export const Comment = (props) => {
+    const {deleteComment} = useContext(CommentContext)
+    const deleteCommentDialog = useRef(null)
+
+    const handleDate = (date) => {
+        if ("timestamp" in props.comment) {
+            return new Date(date).toDateString()
+        }
+    }
+
+    const editDeleteButtons = () => {
+        const currentId = localStorage.getItem("rare_user_id")
+        if (props.comment.user_id === parseInt(currentId)) {
+            return (
+                <div className="commentsButtonContainer">
+                    <button
+                        className="btn commentEditButton"
+                        onClick={() => {
+                            props.history.push(`/comments/edit/${props.comment.post_id}`)
+                        }}>Edit Post</button>
+                    <button
+                        className="btn commentDeleteButton"
+                        onClick={() => {
+                            deleteCommentDialog.current.showModal()
+                        }}>Delete Post</button>
```

Comment:
Everything looks great! I would just change these buttons to either say edit comment and delete comment. Or if you don't put any text and change the class name to ```btn-small fa fa-edit``` and btn-small fa fa-trash``` they will display the edit delete icons